frontend/src/pages/privacy.md

```diff
@@ -0,0 +1,50 @@
+# Privacy Policy
+
+Shields.io is non-tracking and privacy-respecting. This Privacy Policy explains how we handle your data in compliance with the General Data Protection Regulation (GDPR).
+
+## 1. Hosting and Service Providers
+
+We use [fly.io](https://fly.io) for hosting and [CloudFlare](https://www.cloudflare.com) for DNS and CDN services. These third-party providers process requests to deliver and secure our website. Please refer to their privacy policies for more information:
+
+- https://fly.io/legal/privacy-policy/
+- https://www.cloudflare.com/en-gb/privacypolicy/
+
+## 2. Cookies
+
+We do not use any cookies on our website.
+
+## 3. Logs and Data Collection
+
+We do not store any logs of your visits, requests, or other activities on our site.
```

Comment:
We do log requests and emit a bunch of telemetry. I'm wondering whether this paragraph may be a little misleading. Maybe we should combine it with the next one saying we gather telemetry, logs, and errors on requests, but no PII.

Reply:
Thanks for taking a look.

Where do we log requests to?
When we were on heroku, we did used to be able to access a log stream and the last 1,000 requests were retained. This was logging generated by heroku at the load balancer.
Now that we are on fly, the only logs we emit to fly are what the docker container emits, and we don't emit any request logs. AFAIK we don't have access to any logging from fly's load balancer.

When you say "telemetry", are you thinking of the stats we send out to https://metrics.shields.io ?

Reply:
Ah, I had in mind Heroku's logging, and assumed the same was available somewhere in Fly.

Yep, I was thinking of the stats we send to metrics.shields.io. Though not sure they really warrant a mention.